animalfacts.py

```diff
@@ -2249,6 +2263,7 @@ def animalfactsbot(reddit):
     LIZARD_FACTS,
     LOBSTER_FACTS,
     LLAMA_FACTS,
```

Comment:
LLAMA_FACTS = (
'The llama is a South American relative of the camel, though the llama does not have a hump.',
'These sturdy creatures are domestic animals used by the peoples of the Andes Mountains.',